Automattic/tumblr-theme-garden
# Always show active theme in theme browser

### Summary

Fixes #52 

### Solution

In this PR, we show the currently active Tumblr theme (if there is one) at the beginning of the list of themes.

| Natural state | Hover state |
| --- | --- |
| <img width="664" alt="Screenshot 2024-12-18 at 8 55 30 PM" src="https://github.com/user-attachments/assets/389849ab-017d-405b-8f03-a2c2d166128b" /> |  <img width="585" alt="Screenshot 2024-12-18 at 8 55 37 PM" src="https://github.com/user-attachments/assets/dd72db63-a1bc-41d5-84e9-08a76123158d" /> | 

### Testing steps

- change theme to a non-tumblr theme
- visit the theme garden and make sure you can change categories without anything breaking
- activate a tumblr theme
- visit the theme garden and make sure your active theme appears at the top of the list
- enter a search term, like 'window'.
- make sure active theme is not shown in results
